app/web/src/features/communities/events/EventPage.tsx

```diff
@@ -196,6 +197,7 @@ export default function EventPage() {
     <NotFoundPage />
   ) : (
     <>
+      <HtmlMeta title={event?.title} />
```

Comment:
Same comment as in `DiscussionPage` - set `noSuffix` depending on the loading state so we don't get "Couchers.org Beta" twice.

Reply:
fixed by updating the HtmlMeta component